CommandantTests/OptionSpec.swift

```diff
@@ -12,7 +12,7 @@ import Nimble
 import Quick
 import Result
 
-enum NoError {}
+enum NoError: ErrorType {}
```

Comment:
I think we should eventually move this to `Result` so that RAC and this can share it /cc @robrix